core/src/main/java/org/apache/calcite/plan/RelOptUtil.java

```diff
@@ -1755,24 +1755,30 @@ private static void splitJoinCondition(
    * can rewritten in expanded form as
```

Comment:
```suggestion
   * can be rewritten in expanded form as
```

I know it's not part of your PR but since I noticed it, let's take the chance to fix it if you agree on the change